app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,9 @@
   </head>
 
   <body>
+      <% flash.each do |name, msg| -%>
+      <p class="color-blue"><%= msg %></p>
+    <% end -%>
     <%= yield %>
   </body>
 </html>
```

Comment:
- [ ] In users/index.htm.erb, `user.posts_count` should be changed to `user.posts_counter`. Your app shows an error with this line. The same thing with `users/show.htm.erb` file.

- The column name in your schema:
https://github.com/VTY1999/Blog-app/blob/7cbd9b6fa9a6e12de912453759cbee92a3329bc6/db/schema.rb#L51